position/positions.go

```diff
@@ -35,27 +35,27 @@ func AltDotEnd(buff demodel.CharBuffer) (uint, error) {
 
 func TagDotStart(buff demodel.CharBuffer) (uint, error) {
 	if buff.Tagline == nil || len(buff.Tagline.Buffer) == 0 {
-		return 0, Invalid
+		return 0, ErrInvalid
```

Comment:
more context is always useful, but I don't think there's any code that really looks at the error code other than to set a default of 0 or len(buff.Buffer) if something went wrong, so it's probably not worth the effort.